Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrkFilter_cff.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+#------------------------------------------------
+#AlCaReco filtering for HCAL isotrk:
+#------------------------------------------------
+
+from Calibration.HcalAlCaRecoProducers.alcaIsoTracksFilter_cfi import *
+
+import Alignment.CommonAlignmentProducer.AlignmentTrackSelector_cfi
+TkAlIsoProdFilter = Alignment.CommonAlignmentProducer.AlignmentTrackSelector_cfi.AlignmentTrackSelector.clone()
```

Comment:
@bsunanda should't this be called TkAlIsoProd, to be consistent with the following lines?
I get this error when trying to run your PR merged on top of the last IB:
```
 File "/tmp/musich/CMSSW_8_0_X_2016-02-10-2300/python/Calibration/HcalAlCaRecoProducers/ALCARECOHcalCalIsoTrkFilter_cff.py", line 11, in <module>
    TkAlIsoProd.filter = False
NameError: name 'TkAlIsoProd' is not defined
```